### PR TITLE
fixed: QuickInfoMargin updates on each cursor move, even within same word

### PR DESCRIPTION
**Pull request by vasily-kirichenko**

### CODE REVIEW

File: src/FSharpVSPowerTools.Logic/ProjectProvider.fs [hunk 1]
```diff
@@ -13,7 +13,7 @@ type internal ProjectProvider(project: Project,
                               getProjectProvider: Project -> IProjectProvider option, 
                               onChanged: Project -> unit) =
     static let mutable getField = None
-    do Debug.Assert(project <> null, "Input project should be well-formed.")
```
Comment (dungpa) [inline src/FSharpVSPowerTools.Logic/ProjectProvider.fs]: Why inverting this?

Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Logic/ProjectProvider.fs]: OMG. Thanks.

Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Logic/ProjectProvider.fs]: Fixed.

**Comment by dungpa** [comment]
:+1:

**Pull request merged**